tests/components/apple_tv/test_media_player.py

```diff
@@ -0,0 +1,138 @@
+"""Test Apple TV media player."""
+
+import asyncio
+
+from pyatv.const import FeatureName, FeatureState
+from pyatv.interface import (
+    AppleTV,
+    Apps,
+    Audio,
+    BaseService,
+    DeviceInfo,
+    FeatureInfo,
+    Features,
+    Keyboard,
+    Metadata,
+    Power,
+    PushUpdater,
+    RemoteControl,
+    Stream,
+    UserAccounts,
+)
+from pyatv.settings import Settings
+
+from homeassistant.components.apple_tv.media_player import AppleTvMediaPlayer
+from homeassistant.core import HomeAssistant
+
+
+class _MockUnknownFeatures(Features):
+    def get_feature(self, feature_name: FeatureName) -> FeatureInfo:
+        return FeatureInfo(state=FeatureState.Unknown)
```

Comment:
There wasn't any existing tests for this class, so I started one. I'm not very familiar with Python testing best practices - so while this works, and passes (and fails if I mutate the underlying method), I'm totally OK rewriting this if there's some better way to mock these.

Reply:
Please take a look at other tests we have in our repository for some guidance on how we generally set up tests in Home Assistant.

We don't mock/test specific parts of code directly. Instead, we set up Home Assistant with the integration and test against its output.